src/lib/MAVLink/MAVLink.cpp

```diff
@@ -59,6 +414,24 @@ void convert_mavlink_to_crsf_telem(uint8_t *CRSFinBuffer, uint8_t count, Handset
                 crsfgps.p.satellites_in_use = gps_int.satellites_visible;
                 CRSF::SetHeaderAndCrc((uint8_t *)&crsfgps, CRSF_FRAMETYPE_GPS, CRSF_FRAME_SIZE(sizeof(crsf_sensor_gps_t)), CRSF_ADDRESS_CRSF_TRANSMITTER);
                 handset->sendTelemetryToTX((uint8_t *)&crsfgps);
+
+                // send the gps_status message to Yaapu Telemetry Script
+                ap_send_crsf_passthrough_single(0x5002, format_gps_status(gps_int.fix_type, gps_int.alt, gps_int.eph, gps_int.satellites_visible));
+
+
+                // send the home message to Yaapu Telemetry Script
+                uint8_t bearing = 0;
+                uint32_t distance_to_home = 0;
+                if ((home_latitude != 0) && (home_longitude != 0)){
+                    int32_t partial_distance_north = (home_latitude - gps_int.lat) * 111318;
+                    int32_t partial_distance_east = (home_longitude - gps_int.lon) * 111318;
+                    distance_to_home = sqrt((partial_distance_north * partial_distance_north)
```

Comment:
Is there a valid recipe for fixed-point-sqrt on ESP?

This calc seems to go wrong completely.

Reply:
Maybe take a look at this https://github.com/chmike/fpsqrt